examples/sl/_01_sl_zijlstra.py

```diff
@@ -0,0 +1,70 @@
+"""
+SL Zijlstra
+=============================
+
+This example shows how to use the Zijlstra algorithm for calculating stride length
+and how its results compare to the original matlab implementation.
+
+Below we will demonstrate the usage of both methods. But first we load some data.
+
+Example Data
+------------
+We load example data from the lab dataset together with the INDIP reference system.
+We will use a single short-trail from the "HA" participant for this example, as it only contains a single gait sequence.
+The stride length algorithm is designed to work on a single gait sequence at a time.
+
+"""
+
+from mobgap.data import LabExampleDataset
+
+lab_example_data = LabExampleDataset(reference_system="INDIP")
+short_trial: LabExampleDataset = lab_example_data.get_subset(
+    cohort="HA", participant_id="001", test="Test5", trial="Trial2"
+)
+
+# %%
+# SlZijlstra
+# ---------
+# To demonstrate the usage of :class:`~mobgap.algorithm.SlZijlstra` we use the detected initial contacts from the
+# reference system as input.
+reference_ic = short_trial.reference_parameters_relative_to_wb_.ic_list
+reference_ic
+
+# %%
+reference_gs = short_trial.reference_parameters_relative_to_wb_.wb_list
+reference_gs
+
+# %%
+# Then we initialize the algorithm and call the ``calculate`` method.
+# Note that we use the ``sampling_rate_hz`` of the actual data and not the reference system.
+# This is because, the reference parameters are already converted to the data sampling rate.
+from mobgap.sl import SlZijlstra
+from gaitmap.trajectory_reconstruction.orientation_methods import MadgwickAHRS
+
+MS_MS_params = SlZijlstra.PredefinedParameters.tuning_coefficient_MS_MS # default predefined parameters for the biomechanical model
+sl_zijlstra = SlZijlstra(orientation_method = MadgwickAHRS(beta=0.2), **MS_MS_params)
+
+gs_id = reference_gs.index[0]
+data_in_gs = short_trial.data["LowerBack"].iloc[reference_gs.start.iloc[0] : reference_gs.end.iloc[0]]
+ics_in_gs = reference_ic[["ic"]].loc[gs_id] # reference initial contacts
+sensor_height = short_trial.participant_metadata['SensorHeight']/100 # sensor height (cm)
```

Comment:
We should decide, if we want to work in cm or m and then either change the sensorheight loaded from the data or change the expected input for the algorithm.

Reply:
I think we are using m most of the time right? So then it would make sense to change the SensorHeight? Any opinions?

Reply:
That's fine for me! I am not sure about other cases where Sensor Height is needed, but at least in the Zijlstra formula you need sensor height in meters, so we could directly save the value in meters in the participant's metadata